astropy/io/misc/asdf/tags/unit/equivalency.py

```diff
@@ -0,0 +1,47 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+# -*- coding: utf-8 -*-
+
+from astropy.units.equivalencies import Equivalency
+from astropy.units import equivalencies
+from astropy.units.quantity import Quantity
+from asdf.yamlutil import custom_tree_to_tagged_tree
+
+from astropy.io.misc.asdf.types import AstropyAsdfType
+
+
+class EquivalencyType(AstropyAsdfType):
+    name = "unit/equivalency"
+    types = [Equivalency]
+    requires = ['astropy']
+    version = '1.1.0'
+
+    @classmethod
+    def to_tree(cls, equiv, ctx):
+        node = {}
+        if isinstance(equiv, Equivalency):
```

Comment:
Just a style comment, but maybe consider inverting the `if` condition here and raising an error, which will allow you to avoid the `else` below.